Support postcss-load-config 4

In a SvelteKit project:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: svelte-preprocess@4.10.6
npm ERR! Found: postcss-load-config@4.0.0
npm ERR! node_modules/postcss-load-config
npm ERR!   dev postcss-load-config@"4.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peerOptional postcss-load-config@"^2.1.0 || ^3.0.0" from svelte-preprocess@4.10.6
npm ERR! node_modules/svelte-preprocess
npm ERR!   dev svelte-preprocess@"4.10.6" from the root project
npm ERR!   svelte-preprocess@"^4.0.0" from svelte-check@2.7.1
npm ERR!   node_modules/svelte-check
npm ERR!     dev svelte-check@"2.7.1" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: postcss-load-config@3.1.4
npm ERR! node_modules/postcss-load-config
npm ERR!   peerOptional postcss-load-config@"^2.1.0 || ^3.0.0" from svelte-preprocess@4.10.6
npm ERR!   node_modules/svelte-preprocess
npm ERR!     dev svelte-preprocess@"4.10.6" from the root project
npm ERR!     svelte-preprocess@"^4.0.0" from svelte-check@2.7.1
npm ERR!     node_modules/svelte-check
npm ERR!       dev svelte-check@"2.7.1" from the root project
```

--- a/package.json
+++ b/package.json
@@ -118,7 +118,7 @@
     "coffeescript": "^2.5.1",
     "less": "^3.11.3 || ^4.0.0",
     "postcss": "^7 || ^8",
-    "postcss-load-config": "^2.1.0 || ^3.0.0",
+    "postcss-load-config": "^2.1.0 || ^3.0.0 || ^4.0.0",
     "pug": "^3.0.0",
     "sass": "^1.26.8",
     "stylus": "^0.55.0",

